driver-core/src/main/com/mongodb/internal/operation/QueryBatchCursor.java

```diff
@@ -269,7 +269,7 @@ private void getMore() {
                                                              NO_OP_FIELD_NAME_VALIDATOR,
                                                              ReadPreference.primary(),
                                                              CommandResultDocumentCodec.create(decoder, "nextBatch"),
-                                                             connectionSource.getSessionContext()));
+                                                             connectionSource.getSessionContext(), null));
```

Comment:
According to server specification, the one command that doesn't accept server version is getMore.

Reply:
Perhaps add a comment inline to explain its not needed.